### ISSUE TITLE
Thumbnail generation failed in Blender 2.91

### ISSUE THREAD

**Issue by QuantD-VAM** [s-leger/archipack#226]
Archipack version: 1.2.84

Blender version: Blender 2.91

Os version: Windows 10

Description of the issue: I activated ArchiPack in Preferences->Addons and clicked the "Render presets thumbs" button. Thumbnail rendering is shown in a separate window but it will never close, just stops at a certain point (after finishing the floor thumbnails it seems). When I restart Blender an try to add an ArchiPack Window, I get an error message (see attachment) .... ArchiPack is not usable at all ... please advice.

![blender-archipack-error-after-gen-thumnails](https://user-images.githubusercontent.com/77069519/103799439-61e37d00-504b-11eb-8aea-2078163ffb3d.png)

**Comment by s-leger** [comment]
This error is related to custom color management "Filmic" addon.

**Comment by QuantD-VAM** [comment]
> This error is related to custom color management "Filmic" addon.

yes I found the topic here: https://blender.stackexchange.com/questions/170579/i-get-an-error-with-the-archipack

I installed a fresh vanilla version of blender and the Filmic addon is installed OOTB ... so why I am running into this error with a default installation ... and how can I fix it?

**Comment by s-leger** [comment]
Addons are stored in user folder and are not removed by blender's setup, si i guess it still remains here.

**Comment by s-leger** [comment]
Take a look at #137 in comments for a solution.

**Comment by QuantD-VAM** [comment]
I check all that but I don't have filmic installed. Here is what blender gives me when I start it:

`OpenColorIO Error: Error could not read 'C:\PROGRA~4\þ' OCIO profile.
Color management: using fallback mode for management
Color management: Error could not find role data role.
Read prefs: C:\Users\hmand\AppData\Roaming\Blender Foundation\Blender\2.91\config\userpref.blend
Color management: scene view "Filmic" not found, setting default "Standard".
found bundled python: C:\Programme\Blender Foundation\Blender 2.91\2.91\python`

I cannot find Filmic in my addon list and I have no idea where to change the "'C:\PROGRA~4\þ" entry.

**Comment by QuantD-VAM** [comment]
"I check all that" means I changed the config like this:

![add_raw](https://user-images.githubusercontent.com/77069519/103803626-1d5ae000-5051-11eb-9288-e9175eae8f0c.PNG)

but it makes no difference

**Comment by QuantD-VAM** [comment]
I deinstalled blender and removed all folder in the installation path and the user folder .. so all clean. After that I reinstalled the latest blender version again. Same result ... filmic is there again and I have the same issue. To do a Proof of Concept, I added another blender version (PyClone) which resists in a separate folder. Now I can activate ArchiPack and generate the thumbs without any issue. I checked the  datafiles folder and PyClone has the "filmic" folder as well and the "config.ocio" files are identical. It is just that all Blender files are under one Root Folder.

I need to evaluate the ArchiPack Pro version, but before I buy it I need to be sure it is running with a normal Windows 10 installation (again: Blender version: Blender 2.91 .. PyClone is running on 2.91 as well) ... please advice and thanks for your support

**Comment by QuantD-VAM** [comment]
... so final update: since PyClone is running, I just deinstalled blender once more and used the ZIP Version to install the vanilla 2.91 version to a folder (not using the windows installer) ... that works. Since it is an acceptable workaround my last question: is the ArchiPack Pro addon working with this kind of setup? Thanks again ...

**Comment by s-leger** [comment]
Talk with Troy about this limitation (using Raw) as name for ocio profile and looks like the "Raw" name as color space does not make any sense in technical point of view, so it is unlikely to be fixed in Troy's version and also unlikely for the bf to adopt this point of view for backward compatibility reason.

An intermediate solution suggested by using "image.is_data" was proposed, but something is wrong with this setting leading into thumbs pictures rendering way too bright (wich should not be the case) so it was not acceptable as workaround.
The current blender fundation point of view is that such changes are not accepted as we are not required to add support for third party addons basically breaking official blender's releases.

In archipack pro, i was able to implement a dedicated workarround for Troy's filmic addon.
